<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDemo56</class>
 <widget class="QDialog" name="dlgDemo56">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Animacion de un Texto Ingresado con PIL</string>
  </property>
  <widget class="QLabel" name="lblMensaje">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingres el Texto a Animar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMensaje">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>621</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAnimar">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Animar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblGrafico">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>941</width>
     <height>391</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
